packages/wonder-blocks-tokens/src/theme/semantic/semantic-color.ts

```diff
@@ -145,7 +145,7 @@ export const semanticColor = {
             },
 
             disabled: {
-                border: border.primary,
+                border: color.fadedOffBlack32,
```

Comment:
Do we need a changeset file for the tokens update?

Reply:
Added! as a `patch` change as the public API won't change (just a fix).